packages/react/src/themes/teams/components/Icon/svg/icons/callPstn.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import cx from 'classnames'
+import { TeamsProcessedSvgIconSpec } from '../types'
+import { teamsIconClassNames } from 'src/themes/teams/components/Icon/svg'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
+      <g className={cx(teamsIconClassNames.filled, classes.outlinePart)}>
+        <path d="M15 9c.274 0 .523.05.762.154.25.11.462.253.646.436.185.185.328.397.437.647.104.24.155.49.155.763s-.05.523-.154.762a2.065 2.065 0 0 1-1.083 1.083c-.24.104-.49.155-.763.155-.274 0-.524-.05-.763-.155a2.051 2.051 0 0 1-1.082-1.083C13.05 11.524 13 11.274 13 11s.05-.524.155-.763a2.054 2.054 0 0 1 1.082-1.082C14.477 9.05 14.726 9 15 9m0-1c-.411 0-.798.08-1.162.238a3.064 3.064 0 0 0-1.6 1.6C12.079 10.202 12 10.59 12 11s.08.798.238 1.162a3.055 3.055 0 0 0 1.6 1.6c.364.159.751.238 1.162.238.41 0 .798-.08 1.162-.238a3.08 3.08 0 0 0 .954-.646c.272-.272.487-.59.646-.954.159-.364.238-.751.238-1.162s-.08-.798-.238-1.162a3.085 3.085 0 0 0-.646-.954 3.09 3.09 0 0 0-.954-.646A2.883 2.883 0 0 0 15 8zM17.535 24c-.484 0-.966-.248-1.369-.716-.579-.67-.559-1.318-.44-1.743.125-.444.493-.84.86-1.209.412-.41.923-.745 1.192-.91a.976.976 0 0 1 .767-.11l.663.171s1.22-1.853 1.336-2.052l-.459-.687a.982.982 0 0 1-.163-.658c.03-.276.111-.823.325-1.366.158-.404.3-.667.503-.941.373-.51 1.03-.736 1.76-.6 1.306.238 1.476 1.028 1.49 1.357.02.542-.195 3.236-1.567 5.566-1.694 2.873-4.176 3.738-4.453 3.827a1.43 1.43 0 0 1-.445.071zm.759-3.72a5.603 5.603 0 0 0-1 .76c-.189.189-.54.54-.605.77-.05.178-.08.458.235.822.18.21.465.435.749.346.873-.28 2.676-1.309 3.898-3.383 1.256-2.133 1.444-4.62 1.428-5.022-.01-.27-.514-.383-.668-.41-.35-.066-.632.012-.777.21-.152.204-.25.39-.377.714-.17.433-.237.879-.261 1.106.003 0 .374.554.464.696.199.314.21.736.026 1.05-.117.2-1.363 2.095-1.363 2.095a.998.998 0 0 1-1.083.418l-.666-.172z" />
+        <path d="M14.75 20.088c-2.765 0-5.75-1.169-5.75-3.057V16.5a.5.5 0 0 1 .5-.5h9a.5.5 0 1 1 0 1l-8.5.031c0 1.058 2.309 2.057 4.75 2.057l.213-.002c.264-.01.506.21.514.487a.501.501 0 0 1-.487.513l-.24.002z" />
+      </g>
+      <path
+        className={cx(teamsIconClassNames.outline, classes.outlinePart)}
```

Comment:
this should be `className={cx(teamsIconClassNames.filled, classes.filledPart)`

Reply:
sorry but why?

Reply:
because this path of the svg is the filled state and if the classes aren't correct then it won't show up correctly